src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class FigureSupplier {

```

Comment:
The class `FigureSupplier` should not contain an empty line at the beginning of its implementation.